app/django/meridien/issues/views.py

```diff
@@ -15,9 +17,10 @@
 
 # Create your views here.
 @api_view(['GET', 'POST', 'DELETE'])
-@permission_classes([IsAuthenticated])
+@permission_classes([AllowAny])
 @csrf_exempt
 def issue_list(request):
+    permission_classes = (AllowAny,)
```

Comment:
separate this into two views, the last thing i want is ppl to fool around with the api and then we get screwed